week-4/D-methods/exercise-3.js

```diff
@@ -3,26 +3,26 @@ The following code contains syntax errors - try and fix them!
 Once you fix them, run this file, it should output the correct values!
 */
 
-
 var person = {
-    name: "Alice",
-    age: 25,
-    currentAddress: "Glasgow",
-    changeAddress: (newAddress) {
-        currentAddress = newAddress;
-    },
-    celebrateBirthday: function {
-        that.age = that.age + 1;
-    }
+  name: "Alice",
+  age: 25,
+  currentAddress: "Glasgow",
+  changeAddress: function(newAddress) {
+    return (this.currentAddress = newAddress);
```

Comment:
Does this function need to return something at all? What would you use the returned value for?